src/include/distributed/multi_physical_planner.h

```diff
@@ -76,7 +76,8 @@ typedef enum
 	SHARD_FETCH_TASK = 4,
 	MAP_OUTPUT_FETCH_TASK = 5,
 	MERGE_FETCH_TASK = 6,
-	MODIFY_TASK = 7
+	MODIFY_TASK = 7,
+	ROUTER_TASK = 8
```

Comment:
@anarazel says we should keep them separate for now.  Although Router/Modify planned and executed by same parts, there are decision making parts in between that can cause problems.